src/main/java/com/gmail/val59000mc/players/PlayersManager.java

```diff
@@ -824,9 +836,21 @@ public void killOfflineUhcPlayer(UhcPlayer uhcPlayer, @Nullable Location locatio
 
 			if(cfg.getEnableKillEvent()){
 				double reward = cfg.getRewardKillEvent();
-				VaultManager.addMoney(killer, reward);
-				if(!Lang.EVENT_KILL_REWARD.isEmpty()){
-					killer.sendMessage(Lang.EVENT_KILL_REWARD.replace("%money%", ""+reward));
+				if (cfg.getIsEconomyKill()) {
+					VaultManager.addMoney(killer, reward);
+					if (!Lang.EVENT_KILL_REWARD.isEmpty()) {
+						killer.sendMessage(Lang.EVENT_KILL_REWARD.replace("%money%", "" + reward));
+					}
+				} else {
+					ArrayList<String> cmds = cfg.getKillCommands();
+					if (cmds != null) {
+						for (String cmd : cmds) {
+							if (cmd.startsWith("/")) {
+								cmd = cmd.substring(1);
+							}
+							Bukkit.dispatchCommand(Bukkit.getConsoleSender(), cmd.replace("%name%", uhcKiller.getRealName()));
```

Comment:
Put try catch around dispatching the command for `CommandException`.
https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/Bukkit.html#dispatchCommand-org.bukkit.command.CommandSender-java.lang.String-